### PR TITLE
CHK-6326 Adding wallet payment option toggles

### PR DESCRIPTION
**Pull request by boldjoshshea**

### CODE REVIEW

**Comment by JosephLeedy** [comment]
Rather than targeting `main`, we should create a new branch for the [CHK-4297](https://boldapps.atlassian.net/browse/CHK-4297) epic. Let's discuss this tomorrow.

**Review by JosephLeedy — APPROVED** [review]

File: Model/Config.php [hunk 1]
```diff
@@ -316,5 +318,35 @@ public function isExpressPayEnabled(int $websiteId): bool
             ScopeInterface::SCOPE_WEBSITES,
             $websiteId
         );
+    }    
+    
+    /**
+     * Check if Express Pay buttons are enabled.
```
Comment (nicolenorman) [inline Model/Config.php:324]: [nit]: might be good to update the descriptions in this and the next doc block to be specific to these new settings

**Review by nicolenorman — APPROVED** [review]

File: Model/Config.php (hunk 1 above)
Comment (boldjoshshea) [reply Model/Config.php:324]: I agree, we should probably get the design team to come up with updated mocks and add it as a capstone to the feature release

Comment (nicolenorman) [reply Model/Config.php:324]: That's probably a good idea too, for this comment I was just meaning the comment blocks in the config file. Really just a tiny nitpick as it doesn't effect any functionality.

Comment (boldjoshshea) [reply Model/Config.php]: ahhh my mistake, updated!

**Pull request merged**